CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 * Add support for Kafka 3.9.1
 * Fixed MirrorMaker 2 client rack init container override being ignored.
+* Dependency updates (Vert.x 5.0.0, Netty 4.2.1.Final)
```

Comment:
I think we normally do not list these in the operator CHANGELOG file. My take is that the CHANGELOG should be for the users ... and TBH, I think this is meaningless to users.

But feel free to ignore this if you want to 🤷.

Reply:
> We had one line like this for 0.46.0 as well

IIRC that was from some user contribution.

> Said that my thought was just to highlight a big jump as a major release change. We can come back to ignore adding this line with minor Vert.x releases.

The point is, that as a user, you do not care whether it is major or minor. It is just an internal implementation detail.

Reply:
> IIRC that was from some user contribution.

Then we didn't notice and set a precedent :-) But I don't care, we already spent too many lines to discuss on this. I will remove it.